drivers/gles3/shaders/scene.glsl

```diff
@@ -593,11 +593,11 @@ layout(std140) uniform GlobalShaderUniformData { //ubo:1
 	vec4 global_shader_uniforms[MAX_GLOBAL_SHADER_UNIFORMS];
 };
 
-	/* Material Uniforms */
++ /* Material Uniforms */
```

Comment:
```suggestion
/* Material Uniforms */
```

Reply:
Sorry, I happen to upgrade my local llvm version so clang-format version is too high.